docs/api/README.md

```diff
@@ -539,6 +539,32 @@ function* takeOneAtMost() {
 }
 ```
 
+### `flush(channel)`
+
+Creates an effect that instructs the middleware to flush all buffered items from the channel. Flushed items are returned back to the saga, so can they be utilized.
```

Comment:
I mean the word order. I could be wrong, cause I'm not a native speaker) but I thought that verb should go after subject.